packages/tools/src/tools/annotation/LivewireContourTool.ts

```diff
@@ -798,6 +778,89 @@ class LivewireContourTool extends ContourSegmentationBaseTool {
         },
       }
     );
+    return annotation;
+  }
+
+  /**
+   * Clears any in progress edits, mostly used to get rid of accidentally started
+   * contours that happen on clicking not quite the right handle point.
+   */
+  public deleteInProgress(element, config, evt) {
+    if (!this.editData) {
+      // Consider deleting items in progress here by using the element/viewport
+      // to find annotations displayed on the current view.
+      return;
+    }
+    this._endCallback(evt, true);
+  }
+
+  /**
+   * Updates the interpolated annotations with the currently displayed image data,
+   * performing hte livewire on the image data as generated.
+   */
+  public updateInterpolatedAnnotation(
```

Comment:
again, I didn't expect to find this here. Since this is the ContourTool, it doesn't really make sense to interpolate contours outside of the segmentation context.